Formula/a/autopsy.rb

```diff
@@ -20,89 +26,48 @@ class Autopsy < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "5e1ce8b5147639d7737a4013030ee2a059d1b8dd4657554e08e9423a9a6b2f66"
   end
 
-  # Installs prebuilt binaries, broken on arm: https://github.com/Homebrew/homebrew-core/issues/175053
-  deprecate! date: "2024-06-22", because: :does_not_build
```

Comment:
Thanks for working on this, but it looks like the reason we deprecated it still applies:
```
  autopsy
    * Binaries built for a non-native architecture were installed into autopsy's prefix.
      The offending files are:
        /opt/homebrew/Cellar/autopsy/4.21.0/libexec/autopsy/markmckinnon/Export_srudb_macos	(x86_64)
        /opt/homebrew/Cellar/autopsy/4.21.0/libexec/autopsy/markmckinnon/parse_prefetch_macos	(x86_64)
        /opt/homebrew/Cellar/autopsy/4.21.0/libexec/platform/modules/lib/x86_64/libjnidispatch-nb.jnilib	(x86_64)
```

Reply:
@carlocab the formula is still being worked on I was just testing what was the issue with the old formula

Reply:
Sure, no problem! Please feel free to continue working on this.

When you feel that this PR is close to ready, please edit your commit message to read
```
autopsy 4.21.0
```

Thanks!